packages/teleport/src/Apps/useApps.ts

```diff
@@ -34,8 +37,25 @@ export default function useApps() {
     run(() => fetchApps());
   }, [clusterId]);
 
+  const hideAddApp = () => {
+    setAppAddVisible(false);
+    fetchApps();
```

Comment:
I don't think error is handled here

Reply:
If you are referring to `fetchApp` errors, it is handled by the useApp hook and the Apps screen.

Reply:
hm... maybe i'm missing something. So i see this which tells me errors will be handled elsewhere:
```
function fetchApps() {
    return ctx.appService.fetchApps(clusterId).then(setApps);
}

```
I see `useEffect` handling error with useAttempt `run`, but not for `hideAddApp` especially b/c it doesn't return?

Reply:
You are right! Good catch!